src/graphql/public/schema.graphql

```diff
@@ -6,6 +6,7 @@ interface Account {
   id: ID!
   level: AccountLevel!
   limits: AccountLimits!
+  pushNotificationSettings: PushNotificationSettings!
```

Comment:
I don't think we should make it dedicate to pushNotification. I think for instance we will want to activate notification by email also (in a follow up PR).

this could be something like:

```
notifications: {
   push: []
   email: []
}
```

Reply:
If we are planning on doing this soon, It may make sense to do it before we release a version of the app with the notification settings that would cause a breaking change.